appeals/web/src/server/appeals/appeal-details/application-submission-date/application-submission-date.mapper.js

```diff
@@ -0,0 +1,70 @@
+/**
+ * @typedef {import('../appeal-details.types.js').WebAppeal} Appeal
+ */
+import { appealShortReference } from '#lib/appeals-formatter.js';
+
+/**
+ * @param {Appeal} appealData
+ * @param {import('@pins/appeals.api').Appeals.SingleAppellantCaseResponse} appellantCaseData
+ * @param {{day: string, month: string, year: string}} storedSessionData
+ * @returns {PageContent}
+ */
+export const changeApplicationSubmissionDatePage = (
+	appealData,
+	appellantCaseData,
+	storedSessionData
+) => {
+	const shortAppealReference = appealShortReference(appealData.appealReference);
+
+	const day = storedSessionData?.day ?? new Date(appellantCaseData.applicationDate).getDate();
+	const month =
+		storedSessionData?.month ?? new Date(appellantCaseData.applicationDate).getMonth() + 1;
+	const year = storedSessionData?.year ?? new Date(appellantCaseData.applicationDate).getFullYear();
```

Comment:
Should we have `new Date(appellantCaseData.applicationDate)` as a variable as its used a few times?